Python Tutorial Exercises(Week 3)/#3.py

```diff
@@ -0,0 +1,22 @@
+'''
+This script takes an inputs number 's' and stores the number to the 'total_number' list 
+and adds a number to the 'week_number' list to count the weeks. When the 'total_number' 
+is greater than the goal then print the number of weeks that have passed and 
+the final amount he has saved once the goal has been reached.
+'''
+
+g = 1000
+week_number = 0
+total_number = 0
+s = input()
+while s != g:   
```

Comment:
This works but it's a strange comparison to use, as `s` will never equal to `g` regardless of what input was given by the user

Reply:
How to fix this?

Reply:
You should have a look through a few examples online to learn how `while` and `input()` works together